pages/events/[filename].tsx

```diff
@@ -129,6 +129,11 @@ export const getStaticProps = async ({ params }) => {
     (t) => t.node
   );
 
+  const canonical = `${tinaProps.data.global.header.url}events/${params.filename}`;
+  if (tinaProps.data.events.seo) {
```

Comment:
This doesn't check if a URL has been set in Tina already and will always overwrite it
I'd do something like:
if (seo && (seo?.canonical === null || seo?.canonical === "")) {